tests/kuttl/tests/run_simple_playbook/02-cleanup.yaml

```diff
@@ -4,3 +4,10 @@ delete:
 - apiVersion: ansibleee.openstack.org/v1alpha1
   kind: OpenStackAnsibleEE
   name: ansibleee-play
+- apiVersion: ansibleee.openstack.org/v1alpha1
```

Comment:
Yes, the api for job is `batch/v1`. The operator should delete them after deleting the CR. When I wrote the existing test case, I added the pod and job to the errors file, so the test would not pass otherwise

Reply:
In that case I suppose there is no need to keep the explicit deletion steps.